src/data/nuclear-player-bin/PKGBUILD.template.ts

```diff
@@ -1,32 +1,31 @@
 export default `# Maintainer: nukeop <nuclear at gumblert dot tech>
+# Contributor: neurognostic <neurognostic at protonmail dot com>
+
 pkgname=nuclear-player-bin
+_pkgname=\${pkgname%-bin}
 pkgver={{pkgver}}
-repurl=https://github.com/nukeop/nuclear
-commidfull=$(git ls-remote $repurl.git -tags v$pkgver)
-commid=\${commidfull:0:6}
-pkgrel=2
-epoch={{epoch}}
-pkgdesc="A free, multiplatform music player app that streams from multiple sources."
-arch=('x86_64')
-url="http://nuclear.js.org/"
-install=nuclear-player-bin.install
-license=('GPL3')
-depends=('libnotify' 'libappindicator-gtk3' 'libxtst' 'nss' )
+pkgrel=3
+pkgdesc='A free, multiplatform music player app that streams from multiple sources.'
+arch=(x86_64)
+url='http://nuclear.js.org'
+provides=(\$_pkgname)
+license=(GPL3)
+depends=(libnotify libappindicator-gtk3 libxtst nss )
+install=\$pkgname.install
 source=(
-    {{deburl}}
-    nuclear.desktop
```

Comment:
What happened tot he desktop file?

Reply:
It's no longer needed, it's extracted from the `.deb` during `package()` in the `PKGBUILD`. Was there a reason for maintaining a separate `.desktop` file?